app/models/documentation/page.rb

```diff
@@ -45,5 +50,15 @@ def title_with_ancestors
       parents = ancestors.reverse.map(&:title).join(' > ')
       "#{parents} > #{title}"
     end
+
+    def file_extension(file)
+      file.blob.filename.to_s.split('.').last.upcase
+    end
+
+    def allowed_extensions(kind = nil)
+      return ALLOWED_DOCUMENT_EXTENSIONS.join(',') if kind == :document
+
+      ALLOWED_IMAGE_EXTENSIONS.join(',')
```

Comment:
I think we should only return image extension on `kind == :image`, else return an empty array

Reply:
mmm, I Agree...